docs/pages/components/RadioCardGroup.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: RadioCardGroup
+description: A radio group with options styled like cards.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/RadioCardGroup/index.tsx
+---
+
+import { RadioCardGroup, FieldWrap } from '@deque/cauldron-react'
+
+```jsx
+import { RadioCardGroup } from '@deque/cauldron-react'
+```
+
+<Note>
+  Use [FieldWrap](./FieldWrap) to wrap all input components. The `RadioCardGroup` component has been designed specifically to only be used when rendered as a child of this component, and may not be completely accessible if rendered outside of `FieldWrap`.
+</Note>
+
+
+## Examples
+
+```jsx example
+<FieldWrap>
+  <div className="Field__label" id="radio-card-group-label">Do you like coffee?</div>
+  <RadioCardGroup
+    defaultValue="yes"
+    aria-labelledby="radio-card-group-label"
+    radios={[
+      {
+        id: 'yes',
+        value: 'yes',
+        label: 'Yes',
+        cardImg: (
+          <div
+            style={{
+              backgroundColor: 'green',
+              height: 60,
+              width: 60,
+              borderRadius: 60
+            }}
+          ></div>
+        ),
+        cardIcon: 'check-circle'
+      },
+      {
+        id: 'no',
+        value: 'no',
+        label: 'No',
+        disabled: true,
+        cardImg: (
+          <div
+            style={{
+              backgroundColor: 'red',
+              height: 60,
+              width: 60,
+              borderRadius: 60
+            }}
+          ></div>
+        ),
+        cardIcon: 'check-circle'
+      },
+      {
+        id: 'still-yes',
+        value: 'still yes',
+        label: 'Still Yes',
+        cardImg: (
+          <div
+            style={{
+              backgroundColor: 'green',
+              height: 60,
+              width: 60,
+              borderRadius: 60
+            }}
+          ></div>
+        ),
+        cardIcon: 'check-circle'
+      }
+    ]}
+  />
+</FieldWrap>
+```
+
+## Props
+
+<ComponentProps
+  className={true}
+  props={[
+    {
+      name: 'radios',
+      type: 'Array',
+      required: true,
+      description: 'Array of objects containing: label, cardImg, cardIcon (see IconTypes), value (optional), and any supported HTMLInputElement properties'
+    },
+    {
+      name: 'aria-label',
+      type: 'string',
+      description: 'A label for the radio card group is required. This means you must provide either an aria-label or aria-labelledby prop.',
+    },
+    {
+      name: 'aria-labelledby',
+      type: 'string',
+      description: 'A label for the radio card group is required. This means you must provide either an aria-label or aria-labelledby prop.',
+    },
+    {
+      name: 'name',
+      type: 'string'
```

Comment:
what is this `name` here?

Reply:
I was copying over the existing documentation and it wasn't there, my bad.